packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1072,15 +1155,161 @@ test('lib-wallet addOffer for autoswap swap', async t => {
   );
 });
 
+test('lib-wallet can give attestations in offers', async t => {
+  const {
+    wallet,
+    attestationPublicFacet,
+    attestationBrand,
+    attestationIssuer,
+  } = await setupTest({ attestation: true });
+
+  const issuerManager = wallet.getIssuerManager();
+  await issuerManager.add('Attestation', attestationIssuer);
+  await wallet.makeEmptyPurse('Attestation', 'staking purse');
+
+  const invitation = await attestationPublicFacet.makeReturnAttInvitation();
+
+  const rawId = '1593482020370';
+  const id = `unknown#${rawId}`;
+  const offer = {
+    id: rawId,
+    invitation,
+    proposalTemplate: {
+      give: {
+        Attestation: {
+          pursePetname: 'staking purse',
+          value: 30n,
+          type: 'Attestation',
+        },
+      },
+      exit: {
+        onDemand: null,
+      },
+    },
+  };
+
+  await wallet.addOffer(offer);
```

Comment:
odd... I get a type when I hover over `wallet`, but the type of the arg to `addOffer` is `any`. Bummer.